includes/TingSearchCqlDoctor.class.inc

```diff
@@ -0,0 +1,189 @@
+<?php
+
+/**
+ * @file
+ * Class TingSearchCqlDoctor
+ * An attempt to convert non cql search phrases to valid cql.
+ *
+ * examples not valid:
+ * portland (film) => portland and (film)
+ * henning mortensen (f. 1939) => henning and mortensen and "(f. 1939)"
+ * harry and (White night) = harry and "(White night)"
+ *
+ *
+ * valid cql stays the same eg.
+ * dkcclterm.sf=v and dkcclterm.uu=nt and (term.type=bog) not term.literaryForm=fiktion
+ */
+
```

Comment:
Class documentation and file documentation should be separated in two doc blocks.
/**
 * @file
 */ 

/**
 * Class TingSearchCqlDoctor
 */

I known in this case the file only contains the class, but it could have defined namespace, use etc.